apps/src/netsim/NetSimTable.js

```diff
@@ -189,14 +249,16 @@ NetSimTable.prototype.refresh = function (callback) {
 /**
  * Generate throttled refresh function which will generate actual server
  * requests at the maximum given rate no matter how fast it is called. This
- * allows us to coalesce refresh events and reduce server load.
- * @param {number} waitMs - Minimum time (in milliseconds) to wait between
- *        refresh requests to the server.
+ * allows us to coalesce refreshAll events and reduce server load.
  * @returns {function}
```

Comment:
Could we follow closure compiler conventions for documenting function arguments? (https://developers.google.com/closure/compiler/docs/js-for-compiler?csw=1#types)

If the returned function takes no parameters and returns nothing interesting, you could just say "{function()}".